### PR TITLE
HAI-2261 Add user role in user list

### PR DESCRIPTION
**Pull request by umeetiusbaar**
# Description

Add user role/roles to user list in Käyttäjähallinta.

## Type of change

- [ ] Bug fix
- [x] New feature
- [ ] Other

# Instructions for testing

I have only tested this with `yarn start-msw` where it works ok. I do not think that using real backend and API is still operational for adding new contact persons.

# Checklist:

- [x] I have written new tests (if applicable)
- [x] I have ran the tests myself (if applicable)
- [ ] I have made necessary changes to the documentation, link to confluence
      or other location:

# Other relevant info

Please describe here if there is e.g. some requirements for this change or
other info that the tester/user needs to know.

### CODE REVIEW

**Comment by markohaarni** [comment]
Role information is missing in mobile size user cards:
Screenshot 2024-02-01 at 13 14 34

File: src/domain/hanke/accessRights/AccessRightsView.tsx [hunk 1]
```diff
@@ -192,6 +207,16 @@ function AccessRightsView({ hankeUsers, hankeTunnus, hankeName, signedInUser }:
     );
   }
 
+  function getUserRoles(args: HankeUser) {
+    return (
+      <div>
+        {args.roolit.map((role) => (
+          <div>{t(`hankeUsers:roleLabels:${role}`)}</div>
```
Comment (markohaarni) [inline src/domain/hanke/accessRights/AccessRightsView.tsx:214]: According to Figma roles should be separated by comma: https://www.figma.com/file/CSWoUcI2jslxfOgBYgjKuK/Haitaton-HDS-3.0.0?type=design&node-id=1154-13628&mode=design&t=mbEgdg4dADSdU3hI-4.

Comment (umeetiusbaar) [reply src/domain/hanke/accessRights/AccessRightsView.tsx:214]: How did I miss that! But, it also puts "Toteuttaja" before "Rakennuttaja". Is there some logic in this or could they be in alphabetical order (e.g. "Rakennuttaja, Toteuttaja")?

Comment (markohaarni) [reply src/domain/hanke/accessRights/AccessRightsView.tsx:214]: I'm not sure about the order. I don't remember that there would have been any discussion about that.

Comment (umeetiusbaar) [reply src/domain/hanke/accessRights/AccessRightsView.tsx]: According to UX the order should be Omistaja, Rakennuttaja, Toteuttaja, Muu. I will implement this.

Comment (markohaarni) [reply src/domain/hanke/accessRights/AccessRightsView.tsx]: Ok, that's good.

Comment (markohaarni) [reply src/domain/hanke/accessRights/AccessRightsView.tsx]: Looks good and works as expected.

**Pull request merged**